.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,108 @@
+name: Fuzz Testing
+
+on:
+  push:
+    branches:
+      - main
+      - release-*
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
+      - '**/OWNERS'
+      - '**/MAINTAINERS'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
+      - '**/OWNERS'
+      - '**/MAINTAINERS'
+
+env:
+  FUZZ_TIME: "120s"  # Run each fuzzer for 2 minutes
+
+jobs:
+  fuzzing:
+    name: Go Fuzzing Tests
+    runs-on: ubuntu-22.04
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+
+      - name: Set up Go
+        uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5
+        with:
+          go-version: 1.22.x
+          cache: true
+
+      - name: Download dependencies
+        run: go mod download
+
+      - name: Verify fuzz tests exist
+        run: |
+          echo "Checking for fuzz tests..."
+          cd test/fuzz
+          go test -list='Fuzz.*' | grep -E '^Fuzz' || echo "No fuzz tests found"
```

Comment:
What's the point of including `.*` in the regular expression that is sought as a substring?